scripts/openzfs-exceptions.txt

```diff
@@ -86,10 +88,13 @@
 6418 131cc95
 6421 ca0bf58
 6434 472e7c6
+6459 - 			changes don't work properly with ZoL codebase
```

Comment:
I already ported it ([link](https://github.com/gmelikov/zfs/blob/OpenZFS-6459/scripts/cstyle.pl)).

Example of strange output: 
```
./module/zfs/vdev_raidz_math_ssse3.c: 423: stuff after {
./module/zfs/vdev_raidz_math_ssse3.c: 424: non-continuation indented 4 spaces
```

There are good warnings too. I'll add full out here.
[out.txt](https://github.com/zfsonlinux/zfs-buildbot/files/712529/out.txt)